src/components/Skeleton/styles.module.scss

```diff
@@ -4,7 +4,7 @@
   animation: pulse 1s infinite ease-in-out;
   border-radius: $border-radius;
   height: 100%;
```

Comment:
I'd like to see this removed because I don't think it's a good default. It's better to default to the size of text within the container.

Also, when you remove this, `VersionSelect` will no longer need to override it to `height: auto`.

Reply:
If the container is a `p`, then the default size of text is used according to the `p` story.

Removing this rule would break the `div` story though, which I think is valid as one of the constraints in the original issue was to match the dimensions of the containers.

For what it's worth, we need to change the `height` in the `VersionSelect` because what we want to mimic is an HTML `option`.

Reply:
What is the real use case of the div story? I can't think of why we would want that behavior. If we did want it, we could introduce a custom class and set the height explicitly. I don't think it should *default* to this behavior.

Reply:
> What is the real use case of the div story?

That's a good question... I don't know what I was thinking 😞 I updated this code to get rid of the `height: 100%`.